### PR TITLE
OP-705 Add possibility to login by typing the username instead of selecting from a list

### PR DESCRIPTION
**Pull request by nlazzer1**
add section 3.1.36 and update section 3.1.1 in Admin Manual (see OP-705)

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1453,6 +1457,28 @@ By default, ALLOWPRINTOPENEDBILL is set to no.
...
+|===
+|Key |Default Value | Allowed Values
+|USERSLISTLOGIN |no |yes, no
+|===
+
+When Open Hospital is used in multi-user mode, i.e. SINGLEUSER = no, it is possible to choose between two different login modes:
+
+(1) by typing the username directly in a textbox, if USERSLISTLOGIN is set to NO (default):
+
+image:image27.png[UsersListLogin.png,width=305,height=148]
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc:1475]:
```suggestion
image:image27.png[UsersListLogin.png,width=375,height=139]
```

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

Anyway, @nlazzer1 and @dbmalkovsky, don't you think that both image24 and image27 could be set as [width=375,height=139] ?

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -728,7 +729,10 @@ When SINGLEUSER is set to YES (default) the program starts with this user and al
 
 If SINGLEUSER is set to NO the following login window appears when the program is started:
 
-image:image24.png[Login.png,width=305,height=148]
+image:image27.png[UsersListLogin.png,width=305,height=148]
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Why not just change the images to the actual size of image24: 373 X 146?

Comment (nlazzer1) [reply doc_admin/AdminManual.adoc]: I left the same size as the previous image. Where did you find the size 373 X 146?

Comment (mwithi) [reply doc_admin/AdminManual.adoc]: also the previous image had wrong size indeed, you can check it from the [image itself](https://raw.githubusercontent.com/informatici/openhospital-doc/63fe9282b8ee4c30a7c29e5fb5177f75b20a3747/doc_admin/media/image24.png) <-- inspect the html
But what I find is 375x139, not 373x146 @dbmalkovsky

Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc]: @mwithi Yesterday I took a screen shot of the current logon dialog on Linux with JDK 11 and the image parameters are 373 x 146.     Why do we even provide sizes for dialogs that are small and will easily fit on a page? Just let the image render as it is.

Comment (mwithi) [reply doc_admin/AdminManual.adoc]: @dbmalkovsky ok, that could depend on the jdk specific rendering, but for the doc I think we should use the actual sizes of the imagefile uploaded on this repository, and both are 375x139, wdyt?
image

Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc]: Sounds good to me.

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -1453,6 +1457,28 @@ By default, ALLOWPRINTOPENEDBILL is set to no.
...
+
+When Open Hospital is used in multi-user mode, i.e. SINGLEUSER = no, it is possible to choose between two different login modes:
+
+(1) by typing the username directly in a textbox, if USERSLISTLOGIN is set to NO (default):
+
+image:image27.png[UsersListLogin.png,width=305,height=148]
+
+(2) by selecting the username from a list of users, if USERSLISTLOGIN is set to YES:
+
+image:image24.png[Login.png,width=305,height=148]
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
image:image24.png[Login.png,width=375,height=139]
```

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
image:image27.png[UsersListLogin.png,width=375,height=139]
```

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
image:image27.png[UsersListLogin.png,width=375,height=139]
```

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**